src-input/duk_bi_date_windows.c

```diff
@@ -86,10 +86,15 @@ DUK_INTERNAL_DECL duk_int_t duk_bi_date_get_local_tzoffset_windows(duk_double_t
 	ft1.dwLowDateTime = tmp2.LowPart;
 	ft1.dwHighDateTime = tmp2.HighPart;
 	FileTimeToSystemTime((const FILETIME *) &ft1, &st2);
+#if defined(_DURANGO) || defined(_XBOX_ONE)
+	// XboxOne doesn't have 'SystemTimeToTzSpecificLocalTime' API
```

Comment:
Re: platform check, that's correct - I know it's a bit of a hassle sometimes, especially in Date providers which are already platform specific. Ideally there would be a descriptive `DUK_USE_xxx` flag for availability of the API.

I'm fine with that limitation, but it would deserve a short comment on the impact.

A limitation like that is OK IMO if there's no default platform way to get the right behavior. A user can then replace the Date provider if necessary; this is often needed on bare metal targets where there isn't necessarily even an API for current time (let alone a time zone).